UKHO.ADM.Tests/functions/Update-AD.Tests.ps1

```diff
@@ -2,7 +2,15 @@ InModuleScope $mut {
     Describe "Update-AD" {
         Context "with valid cd" {
             Mock Generate-ConfigurationObject       
-            Mock Get-ADChanges { return @{ "CreatedOUs" = 1}}
+            Mock Get-ADChanges { return @{ "CreatedOUs" = 1; SystemColours = @{
```

Comment:
if I dont add the SystemCOlours to the Get-ADChanges return the Update-AD function fails because it uses those values to return the legend of the colours before the run.